### PR TITLE
add Zebra_datepicker v1.9.4 with npm autoupdate config

### LINKED ISSUE #7443: [Request] Add Zebra_Datepicker
**Library name:**  Zebra_Datepicker
**Git repository url:** https://github.com/stefangabos/Zebra_Datepicker
**npm package url(optional):** 
**License(s):** http://www.gnu.org/licenses/lgpl-3.0.txt
**Official homepage:** http://stefangabos.ro/jquery/zebra-datepicker/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7443 
@dannyxx001 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/stefangabos/Zebra_Datepicker
 * Official website (optional, not the repository): https://stefangabos.ro/jquery/zebra-datepicker/
 * NPM package url (optional): https://www.npmjs.com/package/zebra_datepicker
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 197
   - Count of watchers: 15
   - Count of forks: 131
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 13
   - Downloads in the last month: 67

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7443, cc @stefangabos

### CODE REVIEW

**Comment by dannyxx001** [comment]
@pvnr0082t I can't access the homepage by https, can you? @@

**Comment by pvnr0082t** [comment]
@dannyxx001 Uh...yes, I can access it.

**Comment by dannyxx001** [comment]
@PeterDaveHello I think this PR is OK. Please review again, thanks.

File: ajax/libs/Zebra_datepicker/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "iso-8601",
+    "ui"
+  ],
+  "license": "LGPL-3.0",
+  "npmName": "zebra_datepicker",
+  "npmFileMap": [
+    {
+      "basePath": "public",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/Zebra_datepicker/package.json]: @pvnr0082t have you confirmed that `public` folder is for distribution usage?

Comment (pvnr0082t) [inline ajax/libs/Zebra_datepicker/package.json]: @PeterDaveHello No, I haven't. I guessed. I'll confirm it.

**Comment by pvnr0082t** [comment]
@dannyxx001 I have confirmed that ```public``` folder is for distribution usage. Please help me review it again, thank you.

**Comment by dannyxx001** [comment]
@PeterDaveHello I think this PR is OK. Please review again, thanks.

**Pull request merged**